python-sdk/README.md

```diff
@@ -58,6 +58,7 @@ path|status|notebooks|description
 [automl-with-databricks](tutorials/automl-with-databricks)|[![automl-databricks-local-01](https://github.com/Azure/azureml-examples/workflows/automl-databricks-local-01/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/python-sdk-tutorial-automl-databricks-local-01.yml)<br>[![automl-databricks-local-with-deployment](https://github.com/Azure/azureml-examples/workflows/automl-databricks-local-with-deployment/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/python-sdk-tutorial-automl-databricks-local-with-deployment.yml)|[automl-databricks-local-01.ipynb](tutorials/automl-with-databricks/automl-databricks-local-01.ipynb)<br>[automl-databricks-local-with-deployment.ipynb](tutorials/automl-with-databricks/automl-databricks-local-with-deployment.ipynb)|*no description*
 [dataset-uploads](tutorials/dataset-uploads)|[![dataset-uploads](https://github.com/Azure/azureml-examples/workflows/python-sdk-tutorial-dataset-uploads/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/python-sdk-tutorial-dataset-uploads.yml)|[upload_dataframe_register_as_dataset.ipynb](tutorials/dataset-uploads/upload_dataframe_register_as_dataset.ipynb)<br>[upload_directory_create_file_dataset.ipynb](tutorials/dataset-uploads/upload_directory_create_file_dataset.ipynb)|*no description*
 [deploy-local](tutorials/deploy-local)|[![deploy-local](https://github.com/Azure/azureml-examples/workflows/python-sdk-tutorial-deploy-local/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/python-sdk-tutorial-deploy-local.yml)|[1.deploy-local.ipynb](tutorials/deploy-local/1.deploy-local.ipynb)<br>[2.deploy-local-cli.ipynb](tutorials/deploy-local/2.deploy-local-cli.ipynb)|*no description*
+[train-on-low-priority-aml-compute](tutorials/train-on-low-priority-aml-compute)|[![train-on-low-priority-aml-compute](https://github.com/Azure/azureml-examples/workflows/train-on-low-priority-aml-compute/badge.svg?branch=main)](https://github.com/Azure/azureml-examples/actions/workflows/python-sdk-tutorial-train-on-low-priority-aml-compute.yml)|[train-on-low-priority-aml-compute.ipynb](tutorials/train-on-low-priority-aml-compute/train-on-low-priority-aml-compute.ipynb)|Learn how to train models using low-priority AML compute.
```

Comment:
You could add some explanation of what low-priority AML compute is and why it is useful.
Something like:
"Low-priority compute costs significantly less than dedicate compute but jobs on low priority compute can be preempted"

Reply:
Thank you! I used this verbatim!